docs/core/metrics.md

```diff
@@ -156,16 +126,7 @@ You can call `addMetric()` with the same name multiple times. The values will be
 === "addMetric() with the same name"
 
     ```typescript hl_lines="8 10"
-    import { Metrics, MetricUnits } from '@aws-lambda-powertools/metrics';
```

Comment:
Could you please change the line above (L158) to:
```
    ```typescript hl_lines="7 9"
```

this is not an error you introduced but I just noticed that the highlighted lines are not correct, so let's fix it.

Reply:
@dreamorosi , sure, take [look](metrics-code-snippets) .